code/modules/admin/verbs/onlyone.dm

```diff
@@ -1,60 +1,73 @@
-		hijack_objective.owner = H.mind
-		H.mind.objectives += hijack_objective
-
-		H.mind.announce_objectives()
-
-		for (var/obj/item/I in H)
-			if (istype(I, /obj/item/weapon/implant))
-				continue
-			qdel(I)
-
-		H.equip_to_slot_or_del(new /obj/item/clothing/under/kilt/highlander(H), slot_w_uniform)
-		H.equip_to_slot_or_del(new /obj/item/device/radio/headset/heads/captain(H), slot_ears)
-		H.equip_to_slot_or_del(new /obj/item/clothing/head/beret/highlander(H), slot_head)
-		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(H), slot_shoes)
-		H.equip_to_slot_or_del(new /obj/item/weapon/pinpointer(H.loc), slot_l_store)
-
-		var/obj/item/weapon/card/id/W = new(H)
-		W.icon_state = "centcom"
-		W.access = get_all_accesses()
-		W.access += get_all_centcom_access()
-		W.assignment = "Highlander"
-		W.registered_name = H.real_name
-		W.flags |= NODROP
-		W.update_label(H.real_name)
-		H.equip_to_slot_or_del(W, slot_wear_id)
-
-		var/obj/item/weapon/claymore/highlander/H1 = new(H)
-		H.put_in_hands(H1)
-		H1.pickup(H)
+		if(H.stat == DEAD || !(H.client))
+			continue
+		H.make_scottish()
 
 	message_admins("<span class='adminnotice'>[key_name_admin(usr)] used THERE CAN BE ONLY ONE!</span>")
 	log_admin("[key_name(usr)] used THERE CAN BE ONLY ONE.")
 	addtimer(SSshuttle.emergency, "request", 50, FALSE, null, 1)
 
+/mob/living/carbon/human/proc/make_scottish()
+	ticker.mode.traitors += mind
+	mind.special_role = "highlander"
+	dna.species.specflags |= NOGUNS //nice try jackass
+
+	var/datum/objective/steal/steal_objective = new
+	steal_objective.owner = mind
+	steal_objective.set_target(new /datum/objective_item/steal/nukedisc)
+	mind.objectives += steal_objective
+
+	var/datum/objective/hijack/hijack_objective = new
+	hijack_objective.explanation_text = "Escape on the shuttle alone. Ensure that nobody else makes it out."
+	hijack_objective.owner = mind
+	mind.objectives += hijack_objective
+
+	mind.announce_objectives()
+
+	for(var/obj/item/I in src)
+		if(istype(I, /obj/item/weapon/implant))
+			continue
+		qdel(I)
+	equip_to_slot_or_del(new /obj/item/clothing/under/kilt/highlander(src), slot_w_uniform)
+	equip_to_slot_or_del(new /obj/item/device/radio/headset/heads/captain(src), slot_ears)
+	equip_to_slot_or_del(new /obj/item/clothing/head/beret/highlander(src), slot_head)
+	equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(src), slot_shoes)
+	equip_to_slot_or_del(new /obj/item/weapon/pinpointer(src), slot_l_store)
+	var/obj/item/weapon/card/id/W = new(src)
+	W.icon_state = "centcom"
+	W.access = get_all_accesses()
+	W.access += get_all_centcom_access()
+	W.assignment = "Highlander"
+	W.registered_name = real_name
+	W.flags |= NODROP
+	W.update_label(real_name)
+	equip_to_slot_or_del(W, slot_wear_id)
+
+	var/obj/item/weapon/claymore/highlander/H1 = new(src)
+	if(!highlander)
+		H1.admin_spawned = TRUE //To prevent announcing
+	put_in_hands(H1)
+	H1.pickup(src)
+
+	var/obj/item/weapon/bloodcrawl/ANTIWELDER = new(src)

```

Comment:
it's probably just pathed with the bloodcrawl hand cover thing so you can't use the hand